tests/lib/kubernetes/caasp.py

```diff
@@ -34,29 +34,29 @@ def __init__(self, hardware):
         self._clusterpath = os.path.join(hardware.working_dir, 'cluster')
         self._kubeconfig = os.path.join(self.hardware.working_dir,
                                         'admin.conf')
+        self._ssh_agent_auth_sock = os.path.join(hardware.working_dir,
+                                                 'ssh-agent.sock')
         self._ssh_agent()
 
     def _ssh_agent(self):
         try:
-            res = subprocess.run(['ssh-agent'], check=True,
-                                 capture_output=True)
+            res = subprocess.run(
+                ['ssh-agent', '-a', self._ssh_agent_auth_sock],
```

Comment:
Should we add ssh-agent to bindep?

Reply:
hm. only needed for caasp. so we would add a caasp profile to bindep, right?

Reply:
Yeah. We should probably have a profile per CLOUD_PROVIDER

Reply:
you mean HARDWARE_PROVIDER ;)

Reply:
`ssh-agent` is in the ssh client package. I don't think we need to explictly add this to bindep